keadm/cmd/keadm/app/cmd/debug/collect_test.go

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright 2024 The KubeEdge Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
```

Comment:
Please add space line between line 2-3, line 5-6 and  line 6-7.  Same with other files